lib/Agrammon/Web/APIRoutes.pm6

```diff
@@ -80,9 +80,14 @@ sub rest-api-routes (Str $schema, Agrammon::Web::Service $ws) is export {
                         :language(~$language), :format(~$accept), :print-only(~$print-only),
                         :include-filters($include-filters eq 'true'), :all-filters($all-filters eq 'true')
                     );
-                    content ~$accept, supply {
-                        emit $results.encode('utf8');
-                    };
+                    if ~$accept eq 'application/json' {
+                        content ~$accept, @($results);
+                    }
+                    else {
+                        content ~$accept, supply {
+                            emit $results.encode('utf8');
+                        };
```

Comment:
Hm, since this is only doing one `emit` and the calculation work was already done, this isn't adding anything over just passing the text directly to `content`. For the `supply` approach to really be useful we need to do multiple `emit`s as we produce the data, in order to stream it. That means `get-outputs-from-cs` would also need to produce output in pieces.

I figured the case for streaming output would be when we have many datasets and we `emit` the results for each one. However, it seems here we're only doing a single dataset?